### ISSUE TITLE
Second learning rate

### ISSUE THREAD

**Issue by 89douner** [Paper99/SRFBN_CVPR19#53]
After 1 epoch, learning rate was set to 0.000006.

How do I fix it?

![제목 없음4](https://user-images.githubusercontent.com/31752297/79988544-beb4fc80-84e9-11ea-840a-6e6e5b2fd104.png)

**Comment by 89douner** [comment]
I don't know the reason why second learning rate was set to 0.000006 after 1 epoch, but I resolve this problem when I reinstall python, pytorch and other packages. However, this error message, "SRFBB\lib\site-packages\torch\optm\torch\lr_scheduler.py143: UserWarning: The epoch parameter in 'scheduler.step()' was not necessary and is being deprecated where possible. Please use 'scheduler.step()' to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available."

This message is just warning message, so training goes without any problems.

**Comment by Paper99** [comment]
OK, thank you.
